docs/application/dotnet/guides/nui/imageview.md

```diff
@@ -1,39 +1,101 @@
+---
+keyword: image, ImageView, subarea, premultipliedalpha, PixelArea, ResourceReady, Orientation
+---
+
 # ImageView
 
-An image view is a NUI control that displays an image. It is implemented through the `Tizen.NUI.BaseComponents.ImageView` class.
+An `ImageView` is a NUI control that displays an image. It is implemented through the [ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class:
```

Comment:
I am sorry, by mistake I added a colon at the end of the sentence, kindly change it back to a period.

An `ImageView` is a NUI control that displays an image. It is implemented through the [ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class.